lib/arc.d.ts

```diff
@@ -54,29 +56,24 @@ declare module "daostack-arc-js" {
     globalConstraints: Array<ArcContractInfo>;
   }
 
-  // from arc.js
-  export function configure(options: any): Web3;
-  export function getDeployedContracts(): ArcDeployedContracts;
+/********************************
+ * contracts.js
+ */
+export function getDeployedContracts() : ArcDeployedContracts;
 
-  export interface OrganizationSchemeInfo {
-    name: string;
-    address: string;
-    permissions: string;
-  }
-
-  /********************************
-   * Utils
-   */
+/********************************
+ * Utils
+ */
   export interface TransactionLog {
-    address: string;
-    blockHash: string;
-    blockNumber: number;
-    data: string;
-    logIndex: number;
-    topics: Array<string>;
-    transactionHash: string;
-    transactionIndex: number;
-    type: string;
+    address: string,
```

Comment:
Typescript interfaces use semi-colons, not commas

Reply:
weird, i didnt touch that